### ISSUE TITLE
Add 2 accessibility properties from Schema.org

### ISSUE THREAD

**Issue by carakey** [osulp/Scholars-Archive#2710]
### Descriptive summary

We would like to add two schema.org properties to SA works from to support accessibility: Accessibility Feature and Accessibility Summary. These fields will signal to users what actions have been taken to make content accessible. Accessibility Feature will need a local CV that aligns with a Schema.org vocabulary list. After implementing, we should bulk apply an `accessibilityFeature` value of `unknown` to all works which would be updated as files are remediated, serving to track progress. 

### Details

- I have provisionally entered the properties into the [MAP](https://docs.google.com/spreadsheets/d/1koKjV7bjn7v4r5a3gsowEimljHiAwbwuOgjHe7FEtuw/edit?gid=0#gid=0), at the bottom.
- Tentatively: They should appear by default on the metadata web form for each Work Type (i.e. before the "Additional fields" button). 
- We will need to add guidance to the Libguide for depositors.

`accessibilityFeature`

- https://schema.org/accessibilityFeature
- Property label = Accessibility Feature
- Controlled vocabulary
- This property is intended to use the terms listed in the [Schema.org Accessibility Properties for Discoverability Vocabulary](https://w3c.github.io/cg-reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/) spec. However these terms are text values without URIs, so we can set up as a [local CV](https://github.com/osulp/Scholars-Archive/blob/main/config/authorities). I imagine this set up for the metadata web form to work the same way as we have resource_type in SA, where a depositor can do a multi-select. 

`accessibilitySummary` 
 
- https://schema.org/accessibilitySummary
- Property label = Accessibility Summary
- Free text
- This field is used to capture additional details or nuances as needed. 

### Resources

See https://www.w3.org/community/reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209 for more information.

**Comment by lamtu1** [comment]
Hi @carakey, as I am reading the ticket, you mention that `CV` for `:accessibility_feature` does not provide a URL to find the right one. And you opt in the `local_cv`. I was looking through the link you provided, still a bit confuse on what `id and term` to use to build this list? If you can give me an example of the list that would be great!

PS: Also I notice this value is selected automatically as `unknown`, would you want the label to display `unknown` or `blank` on it? 

Thank you!

**Comment by carakey** [comment]
Hi @lamtu1. The `local_cv` should be built using the vocabulary terms listed in the [linked document](https://w3c.github.io/cg-reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/) under section `4.2`. We probably don't need the full list of terms, though. I will touch base with Sagan to narrow down the list to what will be useful for us. Meanwhile for a proof of concept you could start out with the terms under `4.2.1` [structure-and-navigation-terms](https://w3c.github.io/cg-reports/a11y-discov-vocab/CG-FINAL-vocabulary-20241209/#structure-and-navigation-terms). 

The term `unknown` is present in the vocabulary at `4.2.8` so "unknown" should be part of the local CV.

**Comment by carakey** [comment]
@lamtu1 These are the terms to include in the local CV:

annotations
index
pageBreakMarkers
pageNavigation
readingOrder
structuralNavigation
tableOfContents
taggedPDF
alternativeText
audioDescription
closedCaptions
describedMath
longDescription
openCaptions
signLanguage
transcript
synchronizedAudioText
timingControl
unlocked
ChemML
latex
latex-chemistry
MathML
MathML-chemistry
ttsMarkup
highContrastAudio
braille
tactileGraphic
tactileObject
unknown

These are also listed in a [spreadsheet](https://docs.google.com/spreadsheets/d/1Hbyx1zrepZN0UD5o7RwXD0wBHcWsgNuv6TCVpNB0vSg/edit?gid=1964982844#gid=1964982844) if that's easier to read.

**Comment by carakey** [comment]
__QA process:__

- _Created [a new work](https://ir-staging.library.oregonstate.edu/concern/technical_reports/x346d441h) on staging for testing._
- Criteria: Two fields appear on the metadata form for `accessibilityFeature` and `accessibilitySummary` properties, with "feature" as a drop-down and "summary" as a text field.
    - **Pass**. Fields appear on form just before the "additional fields" button and are the correct type. I was able to add multiple values for "feature".
- Criteria: Local CV populates the `accessibilityFeature` dropdown matching the list in comment above.
    **- Almost pass, very small typo problem, "pageNavigatio" missing an "n" at the end**
- Criteria: Values entered in both fields are saved correctly: they appear in the work's RDF metadata (with the expected property URI) and display on the work show page.
    - **Pass**. Show page and NT look correct. 
- Criteria: Properties perform as expected according to MAP attributes
    - **Pass**. Looks good to me!

@lamtu1 I'm sending this back to Ready for that small correction to the CV. Does the rake task to bulk apply "unknown" need a separate ticket?

**Comment by lamtu1** [comment]
@carakey, I will get a quick fix with that typo up and after talking to Ryan Wick, he said we can just do it via console, so no need for a `rake_task` and I can get that updated once it is pass QA.

**Comment by lamtu1** [comment]
@carakey Would you like me to run this over the weekend before the critical deposit period and update any work that does not have an `accessibility_feature` with the `unknown` value?

**Comment by lamtu1** [comment]
`UPDATE`: I was able to manage successfully updated all works on prod with the value of `unknown` as default. There were couple works that has issues with saving as it is due to like old `LDP` is gone or value is not use anymore. But need to double check, but below are the IDs of it:

`p5547w63z, ng451j359, qf85ng57f, 5712mb58m, 8p58pj83s, j6731793q, dr26z2758, tq57nt25v`
